src/Microsoft.Diagnostics.Runtime/src/DataTargets/DacLibrary.cs

```diff
@@ -86,7 +86,12 @@ public DacLibrary(DataTarget dataTarget, IntPtr pClrDataProcess)
             DacDataTarget = new DacDataTargetWrapper(dataTarget);
         }
 
-        public unsafe DacLibrary(DataTarget dataTarget, string dacPath)
+        public DacLibrary(DataTarget dataTarget, string dacPath)
+            : this(dataTarget, dacPath, runtimeBaseAddress: 0)
+        {
+        }
+
+        internal unsafe DacLibrary(DataTarget dataTarget, string dacPath, ulong runtimeBaseAddress)
```

Comment:
This constructor should be public.  Otherwise folks will have no way to create this manually if they so chose to do so.